notifications/events.go

```diff
@@ -0,0 +1,49 @@
+package notifications
+
+const (
+	all            = "*"
+	created        = "created"
+	completed      = "completed"
+	retried        = "retried"
+	failed         = "failed"
+	canceled       = "canceled"
+	expired        = "expired"
+	slashed        = "slashed"
+	separator      = "-"
+	storageDeal    = "storage-deal"
+	storageAuction = "storage-auction"
+	dataRetrieval  = "data-retrieval"
+
+	// All events
+	AllEvents          = all
+	AllCreatedEvents   = all + separator + created
+	AllCompletedEvents = all + separator + completed
+	AllRetriedEvents   = all + separator + retried
+	AllFailedEvents    = all + separator + failed
+	AllCanceledEvents  = all + separator + canceled
+
+	// Storage deal events
+	AllStorageDealEvents      = storageDeal + separator + all
+	StorageDealCreatedEvent   = storageDeal + separator + created
+	StorageDealCompletedEvent = storageDeal + separator + completed
+	StorageDealRetriedEvent   = storageDeal + separator + retried
+	StorageDealFailedEvent    = storageDeal + separator + failed
+	StorageDealCanceledEvent  = storageDeal + separator + canceled
+	StorageDealExpiredEvent   = storageDeal + separator + expired
+	StorageDealSlashedEvent   = storageDeal + separator + slashed
+
+	// Storage auction events
```

Comment:
Auctions aren't relevant in Powergate. We have another project that uses auctions to make filecoin deals. I think at some point there was the idea of integrating that into Powergate, but it won't happen anymore. In fact, we are going to sunset the auctions project in favor of similar functionality that now exist natively in the Filecoin network.

Reply:
ok, I'll remove auction events